### ISSUE TITLE
Geolocation

### ISSUE THREAD

**Issue by mrc-bsllt** [fawmi/vue-google-maps#142]
It would be useful (in case it cannot be done, but it seems to me that it cannot) to implement a geolocation function, which once geolocalised also populates the input with the position found. Has anyone found a solution to this yet?

**Comment by websitevirtuoso** [comment]
Yes. You should do it by your self. It is not part of this lib. I think we can document example to use. but for my case it is very specific

**Comment by mrc-bsllt** [comment]
> Yes. You should do it by your self. It is not part of this lib. I think we can document example to use. but for my case it is very specific

yes, in the end I did it by myself.
I asked because I had once used a library that had a ready-made method for doing it, and it was convenient.
